### PR TITLE
[test] Update mpi_t test on Alps

### PR DESCRIPTION
**Pull request by lucamar**
I have noticed that running in mode `maintenance` the check `mpi_t` is executed on Alps, but the reference files are not up to date: is it still a relevant test, @jgphpc?

### CODE REVIEW

**Review by victorusu — APPROVED** [review]
`lgtm`

**Review by vkarak — CHANGES_REQUESTED** [review]
We need to update also the Daint test and replace `PrgEnv-pgi` with `PrgEnv-nvidia`.

File: cscs-checks/prgenv/mpi_t.py [hunk 1]
```diff
@@ -48,6 +48,10 @@ def set_sanity(self):
                 'control': 'ref/mpit_control_vars_8.1.5.ref',
                 'categories': 'ref/mpit_categories_8.1.5.ref',
             },
+            '8.1.12': {
```
Comment (jgphpc) [inline cscs-checks/prgenv/mpi_t.py:51]: Test cray-mpich/7.7.18 on dom :ok: 
```
[       OK ] (1/4) mpit_check @dom:gpu+PrgEnv-cray
[       OK ] (2/4) mpit_check @dom:gpu+PrgEnv-gnu
[       OK ] (3/4) mpit_check @dom:gpu+PrgEnv-intel
[       OK ] (4/4) mpit_check @dom:gpu+PrgEnv-nvidia
```
remove `PrgEnv-pgi` from the  list please

**Review by jgphpc — COMMENTED** [review]
see comment

**Comment by vkarak** [comment]
> see comment

@jgphpc I don't understand.

**Review by vkarak — APPROVED** [review]

File: cscs-checks/prgenv/mpi_t.py (hunk 1 above)
Comment (lucamar) [reply cscs-checks/prgenv/mpi_t.py:51]: Do you mean to remove `PrgEnv-pgi` from `self.valid_prog_environs` on line 19?

**Comment by lucamar** [comment]
> see comment

@jgphpc You can also add a suggested edit, so it will be quicker to update the request.

**Pull request merged**